packages/typespec-client-generator-core/test/methods/lro.test.ts

```diff
@@ -359,18 +369,48 @@ describe("typespec-client-generator-core: long running operation metadata", () =
           "format",
         );
 
-        const metadata = (method as SdkLroServiceMethod<SdkHttpOperation>).lroMetadata;
+        const metadata = method.lroMetadata;
         ok(metadata);
         strictEqual(metadata.finalStateVia, FinalStateValue.location);
         strictEqual(metadata.finalStep?.kind, "noPollingResult");
 
         const pollingModel = runner.context.sdkPackage.models.find((m) => m.name === "JobState");
         ok(pollingModel);
+        strictEqual(pollingModel.usage & UsageFlags.Input, 0); // polling model should not be input
+        notStrictEqual(pollingModel.usage & UsageFlags.Output, 0); // polling model should be output
         strictEqual(metadata.pollingStep.responseBody, pollingModel);
 
         assert.isUndefined(metadata.finalResponse);
       });
     });
+
+    it("LRO defined in different namespace", async () => {
```

Comment:
for this [issue](https://github.com/Azure/typespec-azure/issues/1752), two namespaces do not have hierarchy. could you change your test to that scenario.

Reply:
updated